Automattic/wp-calypso
# Quick Switch Feature Modal

#### Changes proposed in this Pull Request

* Modal for Quick Switch feature announcement.

<img width="833" alt="Screen Shot 2021-07-22 at 15 03 00" src="https://user-images.githubusercontent.com/1233880/126643568-526bc176-df9c-4a99-8b46-43518c3abc7a.png">

#### Testing instructions

* Go to Calypso live link below
* Go to a Calypso screen that has the Screen Options component rendered on it
* Observe that the modal looks suitable on various browsers and device sizes.
* Close modal, refreshing page should not show the modal again

Fixes https://github.com/Automattic/wp-calypso/issues/54547
